File: tests/android/cryptography/insecure_cipher_3des.java
import javax.crypto.Cipher;

// ruleid: insecure_cipher_3des
Cipher cipher = Cipher.getInstance("3DES/CBC/NoPadding");
// ruleid: insecure_cipher_3des
Cipher cipher2 = Cipher.getInstance("DESede/CBC/NoPadding");
// ruleid: insecure_cipher_3des
Cipher cipher3 = Cipher.getInstance("DESEDEWRAP/CBC/NoPadding");
// ruleid: insecure_cipher_3des
SecretKeyFactory factory = SecretKeyFactory.getInstance("DESede");

File: tests/android/cryptography/insecure_cipher_3des.kt
import javax.crypto.Cipher;

// ruleid: insecure_cipher_3des
val cipher = Cipher.getInstance("3DES/CBC/NoPadding");
// ruleid: insecure_cipher_3des
val cipher2 = Cipher.getInstance("DESede/CBC/NoPadding");
// ruleid: insecure_cipher_3des
val cipher3 = Cipher.getInstance("DESEDEWRAP/CBC/NoPadding");
// ruleid: insecure_cipher_3des
val factory = SecretKeyFactory.getInstance("DESede");